main.go

```diff
@@ -1,43 +1,25 @@
 package main
 
 import (
-	"encoding/json"
+	"GSSTrainingSystem/controllers"
+	"GSSTrainingSystem/services"
 	"fmt"
 	"github.com/gorilla/mux"
 	"net/http"
-	"strconv"
-	"strings"
 )
 
 func main() {
 	router := mux.NewRouter()
 
-	router.Methods("POST").PathPrefix("/courses/").HandlerFunc(postPage)
-	router.Methods("GET").PathPrefix("/courses/").HandlerFunc(getPage)
+	course_service := services.GetCourseService()
+	course_service.ReadInCourses()
 
-	fmt.Println("Server listening on port 8080")
-	http.ListenAndServe(":8080", router)
-}
-
-func postPage(w http.ResponseWriter, r *http.Request) {
-	pageNum, err := strconv.Atoi(strings.TrimPrefix(r.URL.Path, "/courses/"))
-	if err != nil {
-		fmt.Errorf("Could not parse %s to integer. Err: %v", strings.TrimPrefix(r.URL.Path, "/courses/"), err)
-		w.WriteHeader(500)
-		return
-	}
-
-	pageNum++ //increment to next page
-	nextPageNum := strconv.Itoa(pageNum)
+	course_controller := controllers.GetCourseController(course_service)
 
-	fmt.Println("Redirecting to next page...")
+	router.Methods("POST").PathPrefix("/courses/").HandlerFunc(course_controller.PostPage)
+	router.Methods("GET").PathPrefix("/courses/").HandlerFunc(course_controller.GetPage)
+	router.PathPrefix("/public/").Handler(http.FileServer(http.Dir("/public")))
```

Comment:
If this is the 'public' directory, I'd like to change it to 'assets' if thats OK.